### PR TITLE
RIA-7720 Solve vulnerabilities

### PR DESCRIPTION
**Pull request by filippo-possenti-version1**
### JIRA link (if applicable) ###
RIA-7720

### Change description ###
Upgrading libraries to fix suppressions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Review by ThomasKKC — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -187,37 +187,18 @@ dependencyCheck {
...
       entry 'guava'
     }
 
     dependencySet(group: 'commons-fileupload', version: '1.5') {
       entry 'commons-fileupload'
     }
 
-    dependencySet(group: 'org.springframework.security', version: '5.7.5') {
-      entry 'spring-security-crypto'
-      entry 'spring-security-core'
```
Comment (KleoG) [inline build.gradle:211]: is there a reason for the spring security removal? i know the crypto was removed due to the cve - or were the others no long needed?

Comment (filippo-possenti-version1) [reply build.gradle:211]: Entries under `dependencyManagement` are handled by the `io.spring.dependency-management` Gradle plugin, which aims at introducing in Gradle a way of handling dependency versions similar to what Maven used to do.

Essentially, that entry didn't add spring-security but rather overrode its version. By removing it we removed the override, thus allowing the spring version defined in the `org.springframework.boot` plugin to be used instead.

In the past vulnerabilities were found on the previously mapped spring-security version, which forced us to selectively upgrade only spring-security. The updates in this ticket changed the underlying spring version through the `org.springframework.boot` plugin, bumping it up to `2.7.14` , which in turns results in an upgrade of spring-security to `5.7.10`, thus making this override no longer necessary.

Comment (KleoG) [reply build.gradle:211]: Perfect thank you Filippo!

**Review by KleoG — APPROVED** [review]

**Pull request merged**